### ISSUE TITLE
[Bug]: Inheriting API key from collection and passing as param doesn't work

### ISSUE THREAD

**Issue by CliveStHelmet** [httpie/desktop#93]
### What happened?

API keys configured on a collection are not sent along with requests associated with that collection if using the _As param_ option. They appear to work as expected if using the _As header_ option. This applies to API keys retrieved from environment variables, and API keys configured directly within the collection. This happens in both the MacOS and Windows versions of HTTPie (both v2023.3.0).

### Steps to reproduce?

```Markdown
1. Create a new collection
2. Set Auth to `API key`
3. Set type to `As param`
4. Add a key/value pair containing the name of the parameter and the API key to be passed
5. Create a new request
6. Set Auth to `Inherited`
7. Observe in the Preview that no query parameters are added

Can reproduce in MacOS and Windows.
```

### What did you expect to happen?

I expected the query parameters configured on the collection to be inherited by requests associated with the collection.

### Platform

macOS

### HTTPie Desktop version

2023.3.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by naxels** [comment]
Unfortunately I discovered the same bug today. I'm using the macOS version: 2023.3.5 (2023.3.5)

**Comment by jkbrzt** [comment]
We’re working on fix 🛠️

**Comment by jkbrzt** [comment]
✅ Fixed in [v2024.1.1 →](https://github.com/httpie/desktop/releases/tag/v2024.1.1)
